docs/manifest.md

```diff
@@ -248,12 +250,18 @@ Note: The domain of the connect Uri has to be the same as the domain of the [hos
 
 This section is optional.
 
-If the application doesn't need a user-specific runtime configuration, this section can be omitted.
+If the application doesn't need a user-specific runtime configuration, this section can be omitted. It is also ignored for applications with externalInstallationUrl specified.
```

Comment:
make `externalInstallationUrl` a link to the new section below

Reply:
done